media-gfx/nomacs/nomacs-3.10.2.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils xdg-utils
+
+DESCRIPTION="Qt-based image viewer"
+HOMEPAGE="https://nomacs.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="amd64 x86 ~amd64-linux"
```

Comment:
You can't bump with stable keywords, I'm sure `repoman` did complain about that.

Reply:
Fixed.

Sorry, I forgot to edit keywords after copying. I used for checking only `repoman manifest` and `repoman -dx full` withon my local repo-overlay, so there was no warning about it.